modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/view/table/ObjectEntriesTableFDSView.java

```diff
@@ -413,14 +413,16 @@ private String _getFieldName(String businessType, String fieldName) {
 			return StringUtil.replaceLast(fieldName, "creator", "creator.name");
 		}
 
-		if (fieldName.contains(".createDate")) {
-			return StringUtil.replaceLast(
-				fieldName, "createDate", "dateCreated");
-		}
+		if (!fieldName.contains("CPDefinition")) {
```

Comment:
I think is better use `CPDefinition.class.getName()` instead of `"CPDefinition" `, what do you think?

Reply:
@seltong why do we need this condition? I agree with @murilostodolni, I prefer not to rely on the Object Definition name like this.

Reply:
FYI we cannot use the `CPDefinition.class.getName()` here because it is not allowed by the SF.